changelog.md

```diff
@@ -9,6 +9,8 @@
 
 **What's Fixed**
 
+* [Alert]: Added `alert` call for `onClose`'s prop example
```

Comment:
We only include fixes related to our packages and components in the changelog. Documentation and site-related fixes should be excluded from this list. Let's remove those mentions from here.

Reply:
done